src/wwwroot/partials/settings/my-plan.html

```diff
@@ -106,8 +106,8 @@ <h5 class="special uppercase">{{ 'my_plan_pricing_chart' | translate }}</h5>
         </li>
         <li class="strong">
           <h2 id="planPriceBig" class="special plan--price-big">{{vm.currency | translate}} {{vm.leftPlanPrice | number:2 | numberFormat}}</h2>
-          <a ng-class="!vm.disableLeftPlan ? 'show':''" class="hidden-feature-by-display button button--small" href="#/settings/billing?plan={{vm.leftPlanName}}">{{'change_plan' | translate}}</a>
-          <a ng-class="vm.disableLeftPlan ? 'show':''" class="button button--small button--disabled hidden-feature-by-display">{{'change_plan' | translate}}</a>
+          <a ng-class="[!vm.disableLeftPlan ? 'show':'', vm.isValidUpgradeablePlan ? '':'button--disabled']" ng-click="disableNotUpgradable($event)" class="hidden-feature-by-display button button--small" href="#/settings/billing?plan={{vm.leftPlanName}}">{{'change_plan' | translate}}</a>
```

Comment:
Moving conversation to a separate thread.

I usually tend to avoid using logic in the HTML (when it can be done... of course), and try to put things in a more declarative way.
Like a function call, instead of [!vm.disableLeftPlan ? 'show':'', vm.isValidUpgradeablePlan ? '':'button--disabled']
But that is quite personal style... It also make debugging a little easier. Just sharing thoughts.

Reply:
Thanks for sharing your thoughts!, will be considered.